crates/assembly/src/tests.rs

```diff
@@ -2589,6 +2779,35 @@ fn module_alias() -> TestResult {
         r#" help: expected "@", or "adv_map", or "begin", or "const", or "enum", or "proc", or "pub", or "type", or "use", or end of file, or doc comment"#
     );
 
+    Ok(())
+}
+
+#[test]
+#[ignore = "disabled until unused import accuracy is improved"]
+fn module_alias_unused_import() -> TestResult {
```

Comment:
Is the "ignore" statement here still relevant? If so, we should probably create an issue for this (unless we already have one).